### PR TITLE
Add empty Cluster Logs page

### PR DESCRIPTION
**Pull request by mendaomn**
## Description

This PR introduces an inactive Cluster Logs page. It is not reachable by users, and it will be activated in an upcoming PR.

Following PRs will:
- introduce the messages table
- activate the Logs page
- remove the Logs tab from the Cluster page

## Changes

- add Cluster Logs page
- add Log Streams table to the Logs page
- add HelpPanel with temporary copy

## Screenshot

image

## How Has This Been Tested?

- manually, by browsing log streams and testing the property based filter
- unit tests

## PR Quality Checklist

- [x] I added tests to new or existing code
- [ ] I removed hardcoded strings and used [`react-i18next`](https://react.i18next.com/) library ([useTranslation hook](https://react.i18next.com/latest/usetranslation-hook) and/or [Trans component](https://react.i18next.com/latest/trans-component)), see an example [here](https://github.com/aws/aws-parallelcluster-ui/commit/a6f1e2aa46b245b5bf7500a04b83195477a5cfa5)
- [ ] I made sure no sensitive info gets logged at any time in the codebase (see [here](https://cheatsheetseries.owasp.org/cheatsheets/Logging_Cheat_Sheet.html)) (e.g. no user info or details, no stacktraces, etc.)
- [ ] I checked that infrastructure/update_infrastructure.sh runs without any error
- [x] I checked that `npm run build` builds without any error
- [x] I checked that clusters are listed correctly
- [ ] I checked that a new cluster can be created (config is produced and dry run passes)
- [ ] I checked that login and logout work as expected

In order to increase the likelihood of your contribution being accepted, please make sure you have read both the [Contributing Guidelines](../CONTRIBUTING.md) and the [Project Guidelines](../PROJECT_GUIDELINES.md)

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: frontend/src/old-pages/Logs/index.tsx [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    ],
+    [clusterName, t],
+  )
+
+  return (
+    <Layout breadcrumbs={<BreadcrumbGroup items={breadcrumbItems} />}>
+      <ContentLayout
+        header={
+          <Header
+            variant="awsui-h1-sticky"
```
Comment (mtfranchetto) [inline frontend/src/old-pages/Logs/index.tsx:59]: question: does Cloudscape allow sticky headers on non-table headers? My advice is to replace it with a simple h1 variant

**Review by mtfranchetto — COMMENTED** [review]
Overall, great work on such a complex page!

File: frontend/src/old-pages/Logs/LogStreamsTable.tsx [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+        ),
+        noMatch: (
+          <EmptyState
+            title={t('clusterLogs.logStreams.filtering.noMatch.title')}
+            subtitle={t('clusterLogs.logStreams.filtering.noMatch.subtitle')}
+          />
+        ),
+      },
+      pagination: {
+        pageSize: 5,
```
[the remark this answers is not in the document]
Comment (mendaomn) [reply frontend/src/old-pages/Logs/LogStreamsTable.tsx:147]: It is an arbitrary value, chosen with the goal of letting users see both tables in their viewport. We can refine this value as we play with the final implementation. I'll add a clarifying comment to explicit the fact that's an arbitrary choice

File: frontend/src/__tests__/ListClusterLogStreams.test.ts [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+      const data = await ListClusterLogStreams(clusterName)
+
+      expect(data).toEqual([
+        {
+          hostname: 'hostname',
+          instanceId: 'instanceId',
+          logIdentifier: 'logIdentifier',
+          lastEventTimestamp: 'some-timestamp',
+        },
+      ] as LogStreamView[])
```
[the remark this answers is not in the document]
Comment (mendaomn) [reply frontend/src/__tests__/ListClusterLogStreams.test.ts:52]: Love it, thank you

File: frontend/src/old-pages/Logs/LogStreamsTable.tsx [hunk 4]
```diff
@@ -45,10 +45,23 @@ export function LogStreamsTable({clusterName, onLogStreamSelect}: Props) {
     clusterName,
     'headNode',
   ])
-  const {data = [], isLoading} = useQuery('CLUSTER_LOGS', () =>
-    ListClusterLogStreams(clusterName),
-  )
-  const logStreamsToDisplay = data.map(logStream =>
+
+  const [describeClusterQuery, logStreamsQuery] = useQueries([
```
Comment (mtfranchetto) [inline frontend/src/old-pages/Logs/LogStreamsTable.tsx:49]: praise: <3

**Review by mtfranchetto — APPROVED** [review]

**Pull request merged**